orderbook/src/lib.rs

```diff
@@ -44,7 +44,7 @@ pub fn serve_task(
 
     tracing::info!(%metrics_address, "serving metrics");
     metrics_address.set_port(DEFAULT_METRICS_PORT);
-    serve_metrics(registry, metrics_address)
+    serve_metrics(registry, orderbook, metrics_address)
```

Comment:
Maybe I'm missing something obvious, but to me it looks like only the `orderbook` is used for liveliness checks.

Reply:
Yeah, you are right. But the orderbook is the essential object and 'get_solvable_orders' also checks the working of the db. I think it would be sufficient. What else would you recommend to check for the healthiness of the orderbook server?

Reply:
From what I understood from the PR, the liveliness check should also fail if the runloop hasn't been executed in the past X amount of time.

Reply:
The API crate doesn't really have a runloop (or do you mean the maintenance job?). For the solver I implemented the trait on the main driver runloop.

Reply:
THis is what I was missing:

https://github.com/gnosis/gp-v2-services/blob/35a3dcf4558d0fb699b2e75b2fba251d2598a358/solver/src/main.rs#L419

Sorry for the confusion!

Reply:
I do like the idea of also enforcing `run_maintenance` does not get stuck (this might cause a lot of issues as well). But maybe this can happen in a separate PR.